testsuite/MDAnalysisTests/analysis/test_density.py

```diff
@@ -231,86 +334,57 @@ def test_density_from_Universe_userdefn_neqbox(self, universe, tmpdir):
+                delta=self.delta, xdim=1.0, ydim=2.0, zdim=2.0, padding=0.0,
                 gridcenter=self.gridcenters['static_defined'])
 
-        assert len(record) == 2
-        assert str(record[1].message.args[0]) == wmsg
-
-    def test_density_from_Universe_userdefn_ValueErrors(self, universe):
-        import MDAnalysis.analysis.density
+    def test_userdefn_ValueErrors(self, universe):
         # Test len(gridcenter) != 3
         with pytest.raises(ValueError):
-            D = MDAnalysis.analysis.density.density_from_Universe(
+            D = density.density_from_Universe(
                 universe, select=self.selections['static'],
                 delta=self.delta, xdim=10.0, ydim=10.0, zdim=10.0,
                 gridcenter=self.gridcenters['error1'])
         # Test gridcenter includes non-numeric strings
         with pytest.raises(ValueError):
-            D = MDAnalysis.analysis.density.density_from_Universe(
+            D = density.density_from_Universe(
                 universe, select=self.selections['static'],
                 delta=self.delta, xdim=10.0, ydim=10.0, zdim=10.0,
                 gridcenter=self.gridcenters['error2'])
         # Test xdim != int or float
         with pytest.raises(ValueError):
-            D = MDAnalysis.analysis.density.density_from_Universe(
+            D = density.density_from_Universe(
                 universe, select=self.selections['static'],
                 delta=self.delta, xdim="MDAnalysis", ydim=10.0, zdim=10.0,
                 gridcenter=self.gridcenters['static_defined'])
 
-
-class TestGridImport(object):
-
-    @block_import('gridData')
-    def test_absence_griddata(self):
-        sys.modules.pop('MDAnalysis.analysis.density', None)
-        # if gridData package is missing an ImportError should be raised
-        # at the module level of MDAnalysis.analysis.density
-        with pytest.raises(ImportError):
-            import MDAnalysis.analysis.density
-
-    def test_presence_griddata(self):
-        sys.modules.pop('MDAnalysis.analysis.density', None)
-        # no ImportError exception is raised when gridData is properly
-        # imported by MDAnalysis.analysis.density
-
-        # mock gridData in case there are testing scenarios where
-        # it is not available
-        mock = Mock()
-        with patch.dict('sys.modules', {'gridData':mock}):
-            try:
-                import MDAnalysis.analysis.density
-            except ImportError:
-                pytest.fail(msg='''MDAnalysis.analysis.density should not raise
-                             an ImportError if gridData is available.''')
+    def test_has_DeprecationWarning(self, universe):
+        with pytest.warns(DeprecationWarning,
+                          match="will be removed in release 2.0.0"):
+            D = density.density_from_Universe(
+                universe, select=self.selections['static'],
+                delta=self.delta)
```

Comment:
Probably missing something obvious here, but why was `test_presence_griddata` removed?

Reply:
mistake, probably

Reply:
should be brought back

Reply:
I'm assuming that the whole ```TestGridImport``` class should be brought back, i.e. also```test_absence_griddata```, is that correct?

Reply:
@wvandertoorn that's correct.